### PR TITLE
docs: fix in Traefik Hub example - curl is not installed

### PR DESCRIPTION
**Pull request by gander**
docs: fix in Traefik Hub example - curl is not installed in the docker image

### CODE REVIEW

File: docs/hub/traefik.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ services:
     # Enables the development mode, comment the following line to run the hub in prod mode
     command: /usr/bin/caddy run --config /etc/caddy/dev.Caddyfile
     healthcheck:
-      test: ["CMD", "curl", "-f", "https://localhost/healthz"]
+      test: ["CMD", "wget", "--no-verbose", "--tries=1", "--spider", "http://localhost:80/healthz"]
```
Comment (dunglas) [inline docs/hub/traefik.md:35]:
```suggestion
      test: ["CMD", "wget", "--no-verbose", "--tries=1", "--spider", "http://localhost/healthz"]
```

**Review by dunglas — CHANGES_REQUESTED** [review]

**Comment by dunglas** [comment]
Perfect. Can you sign the CLA, please?

**Comment by gander** [comment]
@dunglas accepted

**Comment by dunglas** [comment]
Thanks!

**Pull request merged**